bundles/af-core/src/test/resources/form/dropdown/exporter-dropdown.json

```diff
@@ -20,7 +19,6 @@
     "o"
   ],
   "label": {
-    "richText": false,
     "visible": true,
```

Comment:
why visible is here?

Reply:
same as above, this is test collateral, to verify that if visible is present in crx then sling model honors that.